CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 - Allow optional --ignore-root-warning [#3885](https://github.com/wallabag/wallabag/pull/3885)
 - material: fix left padding of content on medium screens [#3893](https://github.com/wallabag/wallabag/pull/3893)
 - material: hide creation date from card actions on specific sizes [#3894](https://github.com/wallabag/wallabag/pull/3894)
+- Fix slow queries when displaying the main and starred lists
```

Comment:
Why did you put it in the middle of a previous release? 🙂

Reply:
:man_facepalming: 

I can fix that, but hasn't 2.3.8 also been released already? There doesn't seem to be an entry for the next release. I can just leave that out otherwise; I was reminded by the PR template.

Reply:
Yeah the PR template is a bit old, we usually update the changelog when we do the release, you can remove it then